src/map/script.c

```diff
@@ -26567,6 +26592,19 @@ static BUILDIN(rodex_sendmail2)
 			ShowError("script:rodex_sendmail: Missing Item %d amount!\n", (i + 1));
 			return false;
 		}
+
+		if (!data_isint(script_getdata(st, param + 1))) {
+			ShowError("script:rodex_sendmail: Passed amount for item %d is not a number!\n", i + 1);
+			return false;
+		}
+
+		int amount = script_getnum(st, param + 1);
+
+		if (amount < 1 || amount > min(MAX_AMOUNT, SHRT_MAX)) {
+			ShowError("script:rodex_sendmail: Invalid amount %d passed for item %d!\n", amount, i + 1);
```

Comment:
that previous one is under a subfunction
but the main script function `BUILDIN_DEF(rodex_sendmail)` runs from both
https://github.com/HerculesWS/Hercules/blob/32045249e6f91f1d09db2b63ad9eefb04fd1c1e9/src/map/script.c#L27884-L27887
and needs to print actual `buildin_rodex_sendmail` or `buildin_rodex_sendmail_acc` like said before

if can't do then nevermind, this is optional

Reply:
Same here. For the sake of consistency I used the same "function name" as the other error messages in that function.